src/api/movie_api.ts

```diff
@@ -0,0 +1,145 @@
+import {Filters} from 'src/utils/filters'
+import axios from 'axios'
+import {useInfiniteQuery, UseInfiniteQueryResult, useQuery} from 'react-query'
+
+const apiURL = import.meta.env.VITE_API_URL
+const apiKEY = import.meta.env.VITE_API_KEY
+const TORRENT_API_URL = 'https://yts.lt/api/v2/'
+
+export function client(
+  url: string,
+  params?: Record<string, string | number | boolean>,
+): Promise<any> {
+  const config = {
+    baseURL: apiURL,
+    params: {
+      api_key: apiKEY,
+      page: 1,
+      language: 'en-US',
+      ...params,
+    },
+  }
+
+  return axios.get(`${url}`, config).then(({data, status}) => {
+    if (status === 200) {
+      return data
+    } else {
+      return Promise.reject(data)
+    }
+  })
+}
+
+const defaultConfig = {
+  sort_by: Filters.RELEASE_DATE,
+  primary_release_year: 2021,
+  include_adult: false,
+}
+
+function showLoading(result: UseInfiniteQueryResult<any, unknown>) {
```

Comment:
`any` is a foot gun. I suggest typing it to something more explicit.

Reply:
I removed the `<any,unkown>` , since for this function I don't need it.